examples/gio_dbus_register_object/main.rs

```diff
@@ -8,10 +11,48 @@ const EXAMPLE_XML: &str = r#"
         <arg type='s' name='name' direction='in'/>
         <arg type='s' name='greet' direction='out'/>
       </method>
+      <method name='SlowHello'>
+        <arg type='s' name='name' direction='in'/>
+        <arg type='u' name='delay' direction='in'/>
+        <arg type='s' name='greet' direction='out'/>
+      </method>
     </interface>
   </node>
 "#;
 
+#[derive(Debug, glib::Variant)]
+struct Hello {
+    name: String,
+}
+
+#[derive(Debug, glib::Variant)]
+struct SlowHello {
+    name: String,
+    delay: u32,
+}
+
+#[derive(Debug)]
+enum HelloMethod {
+    Hello(Hello),
+    SlowHello(SlowHello),
+}
+
+impl DBusMethodCall for HelloMethod {
+    fn parse_call(
```

Comment:
```suggestion
    fn typed_method_call(
```
as a suggestion for a name?

Reply:
I've changed the method name, but I'm going to leave this thread unresolved, in case e.g. @sdroege has another suggestion.

Other than that this PR is ready as far as I'm concerned.

Reply:
I like this name more :)

Reply:
`typed_method_call()` as compared to the old `method_call()` but with proper types instead of strings and freeform variants

Reply:
But to be clear: for the trait method `parse_call()` sounds better to me